### PR TITLE
Desktop version of the portfolio

### PR DESCRIPTION
**Pull request by ameerhamzakhalid**
I added the Desktop version module in index.html.
I added images required for the Desktop version layout according to the template.
I added the CSS properties required for the Desktop version module.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -643,10 +703,73 @@ footer::after {
...
+
+  .input-full-name,
+  .input-email,
+  textarea {
+    min-width: 327px;
+    width: calc(95vw - 684px);
+  }
+
+  textarea {
+    height: 114px;
```
Comment (akucintavalent) [inline style.css:460]: - [ ] The color of the border of the input fields should be green(ish) when inputting values:

Zrzut ekranu 2022-03-24 o 21 00 40

File: index.html [hunk 2]
```diff
@@ -273,30 +273,35 @@ <h2 class="categorie-title">
...
+          <ul class="form-fields">
+              <li class="input-name">
+                  <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
 
-    
+              </li>
+              <li class="input-full-name">
+                  <input type="text" name="first_name" id="first_name" maxlength="30" placeholder="First name" required>
+                  <input type="text" name="last_name" id="last_name" maxlength="30" placeholder="Last name" required>
+              </li>
```
Comment (akucintavalent) [inline index.html]: - [ ] There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version).

**Review by akucintavalent — CHANGES_REQUESTED** [review]
Hi there, @ameerhamzakhalid 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- GitHub flow is followed :+1:


## Required Changes ♻️

- [ ] Please take care of the responsiveness of your application. When you change the size of the window there should not be any possibility to scroll horizontally.

https://user-images.githubusercontent.com/87897753/159994768-62fe345e-6587-4f0b-accd-002118e325e7.mp4

- [ ] Kindly change the font of the titles of your cards to Poppins. Kindly change their size and boldness.

image

Your design: 
image

- [ ] My recent works section title should have only one line

image

- [ ] Kindly fix the distances in your cards:

Your design:
Zrzut ekranu 2022-03-24 o 20 43 26

Figma template:
Zrzut ekranu 2022-03-24 o 20 45 57

- [ ] There should be a line:

Your design:
image

Figma template:
image

- [ ] Kindly fix the distances here:

image


Figma template:
Zrzut ekranu 2022-03-24 o 20 49 47

- [ ] These links aren't working. They should lead to the corresponding sections of the page.
image

- [ ] Kindly get rid of this blue line:

image

- [ ] In your README, kindly give detailed instructions on how to open the page on a local PC. 

- [ ] Please fix the mobile version of your page and remove the possibility to scroll horizontally. Kindly fix the size of the logo. 

image

- [ ] The empty space after your email should be larger:

Your design:
image

Figma template: 
image

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — COMMENTED** [review]
Hello @ameerhamzakhalid 👋,

_This is an additional review as a result of the second opinion request._

This project is trying to give you experience in working on loosely defined business requirements to create a proper interface. It can be hard to notice little details in the project, But these little details will be the things that separate you as professional developers from beginners. The requested changes may seem small details but they are really important.

You should be careful **not to break** the mobile version while you are adapting your website to the desktop version. 

As stated in the project specification, following the given design principles (typography, margins, placements) are the part of requirements of this project. Please be sure that you implement the requested changes by the @akucintavalent before going any further on this project.

**From The Project Specification** ⬇️ 
> You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

You can always reach me if you have any questions or need further elaboration. 

Keep rocking 🚀!
Sinan Sevgi

**Comment by ameerhamzakhalid** [comment]
I agree with you. But kindly I need to remind that it's my second week, and
too much early to expect that much performance from me

**Comment by sinansevgi** [comment]
> second week, and too much early to expect that much performance from

You have already proven that you can work hard and achieve this by passing the microverse entrance challenges. In order to reach your dreams or goals, you need to push through your limits and past your comfort zone. If you feel that you need more time on a particular topic, you can always choose to repeat the week or you can schedule[ a 1-to-1 meeting](https://microverse.zendesk.com/hc/en-us/articles/360026625973--1-0-2-0-How-can-I-get-help-from-code-reviewers-) with a senior code reviewer to get help.

**Review by CollinsTatang — APPROVED** [review]
Hi @ameerhamzakhalid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**